docs/sources/_index.md

```diff
@@ -23,7 +23,7 @@ form programmable observability **pipelines** for telemetry collection,
 processing, and delivery.
 
 {{% admonition type="note" %}}
-This page focuses mainly on [Flow mode](https://grafana.com/docs/agent/<AGENT VERSION>/flow/), the Terraform-inspired variant of Grafana Agent.
+This page focuses mainly on [Flow mode](https://grafana.com/docs/agent/latest/flow/), the Terraform-inspired variant of Grafana Agent.
```

Comment:
```suggestion
This page focuses mainly on [Flow mode](https://grafana.com/docs/agent/<AGENT_VERSION>/flow/), the Terraform-inspired variant of Grafana Agent.
```

Reply:
Trying with <AGENT_VERSION> (with an underscore instead of a space). We may need to rebuild from the template file.